File: src/main/java/de/twometer/amongus/model/TaskType.java
package de.twometer.amongus.model;

public enum TaskType {
    // General
    AcceptPower,
    DataTransfer,
    FixWiring,
    ClearGarbage,

    // Electrical
    DivertPower,

    // Medbay
    Scan,
    InspectSamples,

    // Weapons
    Shoot,

    // Reactor
    UnlockManifold,
    StartReactor,

    // Sabotages
    FixLights,
    FixComms,
    FixO2Depletion,
    FixMeltdown,

    // Navigation
    StabilizeSteering,
    ChartCourse,

    // Admin
    SwipeCard,

    // Engines
    RefuelEngine,
    EngineMgmt,

    // Shields
    PrimeShields;
}

File: src/main/kotlin/de/twometer/amogus/model/TaskType.kt
package de.twometer.amogus.model

enum class TaskType {
    // Invalid
    Invalid,

    // General
    AcceptPower,
    DataTransfer,
    FixWiring,
    ClearGarbage,

    // Electrical
    DivertPower,

    // Medbay
    Scan,
    InspectSamples,

    // Weapons
    Shoot,

    // Reactor
    UnlockManifold,
    StartReactor,

    // Navigation
    StabilizeSteering,
    ChartCourse,

    // Admin
    SwipeCard,

    // Engines
    RefuelEngine,
    EngineMgmt,

    // Shields
    PrimeShields;
}